docker/server/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:18.12.1
+
+# Create app directore
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+COPY yarn.lock ./
+
+# install ts-node
+RUN npm install -g ts-node
```

Comment:
to avoid running `npm` and conflicting with yarn, we can utilize `yarn dlx` here! 
[example](https://yarnpkg.com/getting-started/migration/#use-yarn-dlx-instead-of-yarn-global)

Reply:
Can you make a follow-up PR to get this working? I had trouble as I need to run `npm` to install yarn berry 🙃